setup.py

```diff
@@ -25,7 +25,6 @@
     extras_require={
         'netifaces': ["netifaces"],
         'asynctest': ["asynctest"],
-        'marshmallow': ["marshmallow"],
```

Comment:
This should appear in the `install_requires` then (as `"marshmallow"`).

Reply:
Oops! It's fixed now.